src/common/memory_desc.hpp

```diff
@@ -87,8 +96,25 @@ const memory_extra_flags_t rnn_s8s8_compensation
         = dnnl_memory_extra_flag_rnn_s8s8_compensation;
 const memory_extra_flags_t compensation_conv_asymmetric_src
         = dnnl_memory_extra_flag_compensation_conv_asymmetric_src;
+// *_conv_asymmetric_src output differs significantly on CPU and GPU so there's
+// need for a separate flag on GPU so that reorders could not mess these two up
+const memory_extra_flags_t compensation_gpu_conv_asymmetric_src
+        = dnnl_memory_extra_flag_compensation_gpu_conv_asymmetric_src;
+const memory_extra_flags_t compensation_gpu_conv_asymmetric_src_bwd
+        = dnnl_memory_extra_flag_compensation_gpu_conv_asymmetric_src_bwd;
+const memory_extra_flags_t compensation_gpu_conv_asymmetric_src_swap
+        = dnnl_memory_extra_flag_compensation_gpu_conv_asymmetric_src_swap;
 } // namespace memory_extra_flags
 
+inline bool check_md_extra_flags(uint64_t flags) {
```

Comment:
This name implies it checks all the flags while in the reality it checks only compensation for gpu. Please make a more narrow name to underline that fact.

Reply:
Renamed to `check_md_extra_flags_compensation_gpu`.